db/db.go

```diff
@@ -24,6 +25,9 @@ var (
 	sshHostPrincipalsTable = []byte("ssh_host_principals")
 )
 
+var crlKey = []byte("crl") //TODO: at the moment we store a single CRL in the database, in a dedicated table.
+//      is this acceptable? probably not....
+
```

Comment:
Can you use the "AuthorityID" as the key? This id is not generally used in all scenarios, but some of them are using it. The id can be read from `a.config.AuthorityConfig.AuthorityID` but if it's empty you can use the default one:

```golang
const DefaultAuthorityID = "00000000-0000-0000-0000-000000000000"
```